pkg/subctl/cmd/utils/restconfig/restconfig.go

```diff
@@ -40,24 +43,82 @@ type RestConfig struct {
 	ClusterName string
 }
 
-func MustGetForClusters(kubeConfigPath string, kubeContexts []string) []RestConfig {
-	configs, err := ForClusters(kubeConfigPath, kubeContexts)
+type RestConfigProducer struct {
```

Comment:
Since "_restconfig_" is the package name.

```suggestion
type Producer struct {
```